scripts/generator/graphql_generator/csharp/Cart.CheckoutAccessors.cs.erb

```diff
@@ -5,6 +5,19 @@ namespace <%= namespace %> {
     using <%= namespace %>.GraphQL;
 
     public partial class Cart {
+
+        /// <summary>
+        /// Retrieve the currency code for the current checkout.
+        /// </summary>
+        /// <exception cref="NoCheckoutException">Calling this accessor before a checkout was created for the Cart. See Cart.GetWebCheckoutLink</exception>
+        public CurrencyCode GetCurrencyCode() {
```

Comment:
I'm looking at the data it's accessing. It's all stuff the end user might want to access. To be honest when I look at it for instance:
```
CurrentCheckout.shippingLine().price()
```
Could be very confusing to the end user to get shipping totals.